.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    name: Build for (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest']
+        python-version: ['3.7', '3.8']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Python info
+        shell: bash -l {0}
+        run: |
+          which python3
+          python3 --version
+      - name: Upgrade pip and install dependencies
+        run: |
+          python3 -m pip install --upgrade pip setuptools
+          python3 -m pip install -r requirements.txt
+          python3 -m pip install -r requirements-dev.txt
+      - name: Check style against standards using prospector
+        run: prospector
+      - name: Check import order
+        run: isort --recursive --check-only virtual-coach-server
+      - name: Run unit tests
```

Comment:
Test coverage is an important metric for monitoring code quality, could you add it after the unit test? then you can send the results to Sonar and show it in README

Reply:
This is added in https://github.com/PerfectFit-project/virtual-coach-server/pull/72